tests/test_tickets.py

```diff
@@ -356,6 +361,34 @@ def test_assign_ticket_to_another_user(self):
         assert self.tc.shirt_size        == DEFAULT_SHIRT_SIZE
         assert self.tc.python_experience == DEFAULT_PYTHON_EXPERIENCE
 
+    def test_assign_ticket_to_another_user_case_insensitive(self):
+        # we need to confirm the order for the tickets to display in the profile
+        self.order.confirm_order(date.today())
+
+        other_user = make_user(self.OTHER_USER_EMAIL)
+
+        response = self.client.post(self.ticket_url, {
+            # This is the only field we're interested in
+            self.prefix('assigned_to'): self.OTHER_USER_EMAIL.upper(),
+
+            # but all the other fields are required.
+            self.prefix('python_experience'): 3,
+            self.prefix('diet'):             'vegetarian',
+            self.prefix('shirt_size'):       'xl',
+        })
+        assert response.status_code == self.VALIDATION_SUCCESSFUL_200
+        self.tc.refresh_from_db()
+        # only care about case insensitive match
+        assert self.tc.assigned_to.lower() == self.OTHER_USER_EMAIL.lower()
+
+        # switch user
+        self.client.login(email=self.OTHER_USER_EMAIL, password='password123')
+        self.tc.refresh_from_db()
+
+        response = self.client.get(self.my_profile_url)
+        self.assertContains(response, 'View your tickets')
+        self.assertContains(response, self.tickets_url)
```

Comment:
I think this tests too little – it does check if there is a link to tickets on the profile page, (which is a correct test), but it doesn't check if a particular ticket is available on the tickets page, which would be what we're really after in this test.

Reply:
Well, the link isn't there if there are no tickets assigned, but the test is simple enough to add.